xml/admin_ceph_upgrade.xml

```diff
@@ -273,14 +273,17 @@
       </para>
      </step>
     </substeps>
-    <warning>
-     <title>Do not Upgrade/Restart the Third &mon;</title>
+    <note>
+     <title>Regarding &mon; upgrade</title>
```

Comment:
```suggestion
     <title>Regarding &mon; Upgrade</title>
```

Reply:
The idea was to have the heading be something like "Note Regarding Ceph Monitor Upgrade", but I'll settle for "Note: Regarding Ceph Monitor Upgrade" ;-)